tests/console/zypper_ar.pm

```diff
@@ -26,9 +26,8 @@ sub run {
             assert_script_run("zypper rr $repourl");
             assert_script_run("zypper rr ftp://" . get_var("REPO_HOST") . "/" . get_var("REPO_$_")) if get_var("REPO_HOST");
             # zdup scenarios might have already $_ added as a repo but disabled
-            assert_script_run("zypper lr -E | grep -w $_ || zypper rr $_");
-            # Skip add repo if already added
-            my $rc = script_run "zypper lr | grep -w $_ || zypper ar -c $repourl $_";
+            assert_script_run("zypper rr $_");
+            my $rc = script_run "zypper ar -c $repourl $_";
             # treat OSS error as test failure, others can be just recorded
```

Comment:
I wonder if this is obsolete now, it should never fail

Reply:
Ok lets get this merged, and I'll do the follow up

Reply:
Created: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/23089 as the follow up